spec/models/profile_spec.rb

```diff
@@ -1016,14 +1016,14 @@
   # TODO: related: should we test against an active profile here?
   describe '#deactivate_for_fraud_review' do
     it 'sets fraud_review_pending to true and sets fraud_pending_reason' do
-      profile = create(:profile, user: user)
+      profile = create(:profile, :in_person_verification_pending, user: user)
 
       expect(profile.activated_at).to be_nil
       expect(profile.active).to eq(false) # ???
       expect(profile.deactivation_reason).to be_nil
       expect(profile.fraud_review_pending?).to eq(false) # to change
       expect(profile.gpo_verification_pending_at).to be_nil
-      expect(profile.in_person_verification_pending_at).to be_nil
+      expect(profile.in_person_verification_pending_at).not_to be_nil # to change
```

Comment:
👍 or 👎 on this comment (and the one below on 1038)? I was following the pattern established on 1024, but am of half a mind to think that it could be misread as meaning that this behavior is subject to change in a future sprint, rather than that we test that this value has been changed.

(And is it bad that most of the test is devoted to assertions that don't change and we need comments to highlight what does?) 🤔

Reply:
I vote 👍 - I like when things like this are pointed out like this when there is a lot of data. Also, because it is in a test file- I did not interrupt as change in a future sprint.

Reply:
if we include this comment I would include when it should be changed so we have an indicator as to when the comment should be removed. 
I lean toward not including the comment on line #1038

Reply:
I think we can use RSpec to write specs that actually highlight what should change, and get better error messages too

```ruby
expect { profile.deactivate_for_fraud_review }.to(
    change { profile.reload.in_person_verification_pending_at }.to(nil)
    .and(change { profile.reload.fraud_review_pending?).from(false).to_true)),
)
```

See https://stackoverflow.com/a/29389223